scripts/fix_broken_files.py

```
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+"""
+Fix files that were broken by aggressive A2A cleanup
+"""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)